docs/content/get-started/get-playing.md

```diff
@@ -28,11 +28,13 @@ This generates a short link to share with anyone who wants to join in for the du
 
 **For more of a challenge, let’s now invite 200 enemies you can fight it out against!**
 
-These enemies will be Unity Clients running in the cloud, mimicking real players of your game from a behavior and load perspective. Their behavior is currently quite simple, but you could extended them to include additional gameplay features.
+These enemies are simulated players; they are `UnityClient` client-worker instances running in the cloud, mimicking real players of your game from a behavior and load perspective. Their behavior is currently quite simple, but you could extend them to include additional gameplay features.
 
-In fact, as far as SpatialOS is concerned, these Unity Clients are indistinguishable from real players, so this is a good approach for regular scale testing.
+In fact, as far as SpatialOS is concerned, these simulated players are indistinguishable from real players; both simulated players and real players are just `UnityClient` client-worker instances to SpatialOS, so this is a good approach for regular scale testing.
 
-To get the legion of enemies started, we will use [Worker Flags]({{urlRoot}}/content/glossary.md#worker-flags), which you can find from your web Console's Deployment Overview page:
+The simulated players are hosted in a separate deployment to ensure that they do not share resources with your `UnityGameLogic` server-worker instances. You can find this deployment by returning to your SpatialOS [Console](https://console.improbable.io/projects) and selecting the deployment with the `_sim_players` suffix.
+
+To get the legion of enemies started, we use [Worker Flags]({{urlRoot}}/content/glossary.md#worker-flags), which you can find from your SpatialOS Console's Deployment Overview page:
```

Comment:
should this be rephrased to a "you"-version instead of using "we"? it's a slightly confusing transition seeing as we use you everywhere else